robots/issue-creator/creator/creator.go

```diff
@@ -163,7 +163,7 @@ func (c *IssueCreator) initialize() error {
 	}
 	token := strings.TrimSpace(string(b))
 
-	c.client = RepoClient(githubClient{ghclient.NewClient(token, c.dryRun)})
+	c.client = RepoClient(githubClient{ghclient.NewClient("https://api.github.com", token, c.dryRun)})
```

Comment:
Here `https://api.github.com` is still hardcoded and it is outside of prow package, suggestion welcomed. Otherwise I'll take this as a TODO.

Reply:
One way might be keeping the old `ghclient.NewClient` interface and add a new one like `ghclient.NewClientWithEndpoint(..)`.